diff --git a/node_modules/send/index.js b/node_modules/send/index.js
index 89afd7e..f70104a 100644
--- a/node_modules/send/index.js
+++ b/node_modules/send/index.js
@@ -789,7 +789,12 @@ SendStream.prototype.stream = function stream (path, options) {
   // pipe
   var stream = fs.createReadStream(path, options)
   this.emit('stream', stream)
-  stream.pipe(res)
+  if (options.transformStream) {
+    debug('apply transform stream between file and res streams')
+    stream.pipe(options.transformStream).pipe(res)
+  } else {
+    stream.pipe(res)
+  }
 
   // cleanup
   function cleanup () {
